validator/client/validator.go

```diff
@@ -652,6 +652,7 @@ func (v *validator) logDuties(slot types.Slot, duties []*ethpb.DutiesResponse_Du
 		} else {
 			attesterKeys[duty.AttesterSlot-slotOffset] = append(attesterKeys[duty.AttesterSlot-slotOffset], validatorKey)
 			totalAttestingKeys++
+			ValidatorNextAttestationSlotGaugeVec.WithLabelValues(validatorNotTruncatedKey).Set(float64(duty.AttesterSlot))
```

Comment:
This needs to be wrapepd with if v.emitAccountMetrics

```suggestion
			if v.emitAccountMetrics {
			ValidatorNextAttestationSlotGaugeVec.WithLabelValues(validatorNotTruncatedKey).Set(float64(duty.AttesterSlot))
			}
```

Please excuse my formatting, github doesn't do a great job with formatting in suggestions. Please run gofmt after accepting these changes.

Reply:
Fixed as requested.